internal/signaling/peer.go

```diff
@@ -247,6 +243,7 @@ func (p *Peer) HandleHelloPacket(ctx context.Context, packet HelloPacket) error
 		if inLobby {
 			logger.Info("peer rejoining lobby", zap.String("game", p.Game), zap.String("peer", p.ID), zap.String("lobby", p.Lobby))
 			p.Lobby = lobby
+			p.store.Subscribe(ctx, p.Game+p.Lobby, p.ForwardMessage)
 			p.store.Subscribe(ctx, p.Game+p.Lobby+p.ID, p.ForwardMessage)
```

Comment:
Suggestion (non blocking): Is it worth making this one call? e.g. `Subscribe(context.Context, cb func(), topics ...string)`

Reply:
I have made this change now. The nice thing is that the Subscribe happens in a single mutex lock. And that we only have one goroutine waiting to unsubscribe. But I think the Subscribe call itself is slightly less readable.